networkx/algorithms/approximation/distance_measures.py

```diff
@@ -39,6 +39,18 @@ def diameter(G, seed=None):
     d : integer
        Lower Bound on the Diameter of G
 
+    Examples
+    --------
+    >>> from networkx.algorithms.approximation.distance_measures import diameter
```

Comment:
We want users to call `nx.diameter`, not import the function from this (very deep) package!